### PR TITLE
Added timing to hb_mc_tile_groups_execute() 

### PR DESCRIPTION
**Pull request by bornaehsani**
function to print the total execution time in both cycles and seconds.
Currently timing results seem to be wrong. Do NOT merge until discussed.

### CODE REVIEW

File: libraries/bsg_manycore_cuda.h [hunk 1]
```diff
@@ -5,8 +5,10 @@
 
 #ifdef __cplusplus
 #include <cstdint>
+#include <ctime>
 #else
 #include <stdint.h>
+#include <time.h>
```
Comment (tommydcjung) [inline libraries/bsg_manycore_cuda.h]: how about making a stopwatch class like this?
```
#include <chrono>
#include <stdexcept>
#include "stopwatch.h"
using namespace std;
using namespace std::chrono;


/*
 *	default constructor.
 */
Stopwatch::Stopwatch()
{
	_started = false;
}

/*
 * 	start the stopwatch.
 */
void Stopwatch::start()
{
	if (_started)
	{
		throw runtime_error("Stopwatch has already started.");
	}
	_started = true;
	_start = high_resolution_clock::now();
}

/*
 * 	stop the stopwatch and return the duration.
 * 	@return amount of time elapsed in seconds.
 */
double Stopwatch::stop()
{
	if (!_started)
	{
		throw runtime_error("Stopwatch has not started.");		
	}

	duration<double> diff = high_resolution_clock::now() - _start;
	_started = false;	
	return diff.count(); 
}
```

Comment (tommydcjung) [reply libraries/bsg_manycore_cuda.h]: just an example of an interface.

Comment (mrutt92) [reply libraries/bsg_manycore_cuda.h]: except we should make it C friendly.

**Comment by drichmond** [comment]
I feel like this might be superseded by the new vanilla stats parser. Is that true?

**Comment by bornaehsani** [comment]
True. Thanks for pointing it out! Addressed in PR # 198 in bsg_manycore.

**Pull request closed without merging**